README.md

```diff
@@ -139,7 +139,7 @@ if they complete without error, you should be in good shape to start using Julia
 
 You can read about [getting started](https://docs.julialang.org/en/stable/manual/getting-started/) in the manual.
 
-If you are building a Julia package for distribution on Linux, OS X,
+If you are building a Julia package for distribution on Linux, MacOS,
```

Comment:
`MacOS` -> `macOS` here and on the other places?

Reply:
Feels wrong to do so since it is a proper noun.

Reply:
It is called `macOS` though, e.g. https://en.wikipedia.org/wiki/MacOS